spec/acceptance/mod_passenger_spec.rb

```diff
@@ -10,10 +10,7 @@
     case fact('operatingsystem')
     when 'Ubuntu'
       case fact('lsbdistrelease')
-      when '14.04'
-        passenger_root = '/usr/lib/ruby/vendor_ruby/phusion_passenger/locations.ini'
-        passenger_default_ruby = '/usr/bin/ruby'
-      when '16.04'
+      when '14.04', '16.04', '18.04'
         passenger_root = '/usr/lib/ruby/vendor_ruby/phusion_passenger/locations.ini'
         passenger_default_ruby = '/usr/bin/ruby'
       else
```

Comment:
These versions are no longer supported so you can simplify the whole block. Perhaps Debian deserves the same treatment.

Reply:
cleaned it up